README.md

```diff
@@ -83,6 +83,30 @@ RUSTFLAGS="--cfg tokio_unstable" AWS_PROFILE=<profile-name> RUST_LOG=debug cargo
 Follow installation instructions for [tokio-console](https://github.com/tokio-rs/console) and then run the
 example with `tokio-console` running.
 
+#### End to end tests
+
+End-to-end tests are available in tests/e2e_transfer_test.rs to validate functionality against a real S3 server.
+To run these tests:
+
+1. Enable end-to-end testing by setting `RUSTFLAGS="--cfg e2e_test"`
+2. Set up a test bucket by following the aws-c-s3 test helper instructions [here](https://github.com/awslabs/aws-c-s3/blob/main/tests/test_helper/README.md)
+
+Sample commands:
+
+```sh
+# Install required Python dependency
+pip3 install boto3
+
+# Configure test environment
+export CRT_S3_TEST_BUCKET_NAME=<your-bucket-name>
```

Comment:
Again, probably best to have different buckets for CRT and Rust transfer manager.